core/src/test/java/com/dtolabs/rundeck/core/common/TestNodesYamlGenerator.java

```diff
@@ -228,7 +228,7 @@ public void testGenerate() throws Exception {
     }
 
     private Map parseYamlMap(String outputString) {
-        Yaml yaml = new Yaml(new SafeConstructor(new LoaderOptions()));
+        Yaml yaml = new Yaml(new LoaderOptions());
```

Comment:
not sure why this change was clobbered, I think since we already merged forward compatible changes for snakeyaml2
```suggestion
        Yaml yaml = new Yaml(new SafeConstructor(new LoaderOptions()));
```

Reply:
Safeconstructor was needed in 1.x as a mitigation for the cve. Snakeyaml 2 uses safeconstructor as default so I went for the cleaner option. But in the end both forms are exactly the same.